runrole

```diff
@@ -33,4 +33,5 @@ if [[ $# -eq 0 ]] ; then
     exit 0
 fi
 
-ansible-playbook -i $INVENTORY $PLAYBOOK ${ARGS} --connection=local -e "role_to_run=$1"
+ansible -m setup -i $INVENTORY localhost --connection=local -e ansible_python_interpreter=/usr/bin/python3 | grep python
+ansible-playbook -i $INVENTORY $PLAYBOOK --connection=local -e ansible_python_interpreter=/usr/bin/python3
```

Comment:
Why is `-e "role_to_run=$1"` being removed from `runrole` ?

Reply:
corrected below